spark/src/main/java/io/ddf/spark/DelegatingDDFManager.java

```diff
@@ -0,0 +1,63 @@
+package io.ddf.spark;
```

Comment:
Is it true that the purpose of this DelegatingDDFManager is for usage in test-cases only?